src/test/java/com/ksm/wstbackoffice/controller/AddressControllerIT.java

```diff
@@ -0,0 +1,70 @@
+package com.ksm.wstbackoffice.controller;
+
+import com.ksm.wstbackoffice.endpoint.EndPoint;
+import io.restassured.http.ContentType;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlGroup;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.Matchers.equalTo;
+
+@SqlGroup({
+        @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/addressInsert.sql"),
+        @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDrop.sql")
```

Comment:
Please rename addressDrop to what you are doing in that file. you are not dropping the table.

Reply:
done